app/models/micropost.rb

```diff
@@ -0,0 +1,21 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
```

Comment:
belongs_to có tác dụng gì em

Reply:
Dạ belongs_to này là mình đang khai báo 1 phía relation của Micropost với User ạ, trong trường hợp này là quan hệ many to one ạ.